server/protocol.go

```diff
@@ -99,13 +99,16 @@ func parseAttestEK(ek *attest.EK) (EK, error) {
 	if reflect.TypeOf(ek.Public).String() != "*rsa.PublicKey" {
 		return EK{}, errors.New("Invalid key type:" + reflect.TypeOf(ek.Public).String())
 	}
-	var c = ek.Certificate.Raw
+	var c []byte = make([]byte, 1)
```

Comment:
Add a comment to explain why we need this trick.

Reply:
Idk why I created a byte slice of length 1, just checked and an empty byte slice works perfectly.
Fixed